x/stakeibc/keeper/icqcallbacks_withdrawal_reward_balance_test.go

```diff
@@ -102,6 +110,25 @@ func (s *KeeperTestSuite) TestWithdrawalRewardBalanceCallback_SuccessfulNoTransf
 	})
 }
 
+// Verify that if the amount returned by the ICQ response is less than the min_swap_amount, no transfer happens
+func (s *KeeperTestSuite) TestWithdrawalRewardBalanceCallback_SuccessfulWithRebate() {
```

Comment:
Nit: we should add tests for the rebate = 0 and rebate = 1 edge cases

Reply:
added more test cases to `CheckForCommunityPoolRebate`